src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1357,16 +1378,18 @@ public String getDisplayName() {
             return GitSCMExtensionDescriptor.all();
         }
 
+        @SuppressFBWarnings(value="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE", justification = "Plugin depends on git-client which always provides implementations")
         public boolean showGitToolOptions() {
-            return Jenkins.getInstance().getDescriptorByType(GitTool.DescriptorImpl.class).getInstallations().length>1;
+            return Jenkins.getActiveInstance().getDescriptorByType(GitTool.DescriptorImpl.class).getInstallations().length>1;
```

Comment:
That's not really much better though, and you'd have to completely break plugin dependency handling to encounter it anyway.